### PR TITLE
Replace ''return'' with lambda expression ''=>''.

### PR DESCRIPTION
**Pull request by GeorgiPopovIT**
I think it's a simple and clearer expression.

### CODE REVIEW

**Comment by Tratcher** [comment]
@GeorgiPopovIT, the compiler disagrees with one of these changes 😀. Can you fix or revert that part?

Note we don't normally accept PRs that only change code style.

**Comment by GeorgiPopovIT** [comment]
@Tratcher, thanks for feedback.

File: src/Http/Authentication.Abstractions/src/AuthenticationProperties.cs [hunk 1]
```diff
@@ -142,7 +142,6 @@ public void SetString(string key, string? value)
             Items.Remove(key);
         }
     }
```
Comment (Tratcher) [inline src/Http/Authentication.Abstractions/src/AuthenticationProperties.cs:144]:
```suggestion
    }

```

**Review by Tratcher — APPROVED** [review]

**Comment by GeorgiPopovIT** [comment]
@Tratcher  thank you for your code review!

**Pull request merged**